bmlt-meeting-list.php

```diff
@@ -22,9 +22,57 @@ class Bread {
 		var $mpdf = '';
 		var $meeting_count = 0;
 		var $formats_used = '';
+		var $formats_by_key = '';
 		var $formats_spanish = '';
 		var $formats_all = '';
-		
+		var $meeting_fields = array (
```

Comment:
Might it make sense to query the root server for available fields rather than hardcode it?

For example: https://example.org/main_server/client_interface/json/?switcher=GetFieldKeys

Reply:
I thought this was a change you made in 1.10, that the fields retrieved should be limited to the ones really used, for performance reasons.   But if performance is OK getting all meeting data, that's a good idea.

Reply:
Yes, this was.  I wasn't sure as to how you how were pulling "other fields".  The performance is definitely important for larger schedules.  You can leave this then.